src/wpt/@types/Key.ts

```diff
@@ -0,0 +1,257 @@
+/**
+ * @url https://github.com/WPO-Foundation/wptagent/blob/master/internal/support/keys.json
+ */
+
+export type Key =
```

Comment:
Should we let this be inferred automatically?

```ts
import keys from 'https://raw.githubusercontent.com/WPO-Foundation/wptagent/22.01/internal/support/keys.json' assert {
	type: 'json',
};

export type Key = keyof typeof keys;
```

I’ve picked v22.01 here, but happy to stick with the `master` branch.